[Changelog] 25.1.0_cycle5.mdx (#521)
seqeralabs/docs: changelog/seqera-cloud/v24.2.0_cycle23.mdx

### Bug fixes

- **Data Explorer**

>     - Enable task working directory navigation using Data Explorer in personal workspaces by adding routing.

Reviewer: ```suggestion
    - Add routing to enable task working directory navigation using Data Explorer in personal workspaces.
```
